### PR TITLE
OCSP: Add return value checks.

### PR DESCRIPTION
**Pull request by paulidale**
The calls are unlikely to fail but better checking their return than not.

Also added some blank lines to separate declarations from code.

### CODE REVIEW

File: crypto/ocsp/ocsp_vfy.c [hunk 1]
```diff
@@ -301,8 +304,11 @@ static int ocsp_match_issuerid(X509 *cert, OCSP_CERTID *cid,
             return -1;
         if (memcmp(md, cid->issuerNameHash.data, mdlen))
             return 0;
-        X509_pubkey_digest(cert, dgst, md, NULL);
-        if (memcmp(md, cid->issuerKeyHash.data, mdlen))
+        if (!X509_pubkey_digest(cert, dgst, md, NULL)) {
+            OCSPerr(0, OCSP_R_DIGEST_ERR);
+            return -1;
```
Comment (kroeckx) [inline crypto/ocsp/ocsp_vfy.c:309]: Please at least document return values.

Comment (paulidale) [reply crypto/ocsp/ocsp_vfy.c:309]: I wish I knew what they meant :)

Comment (kroeckx) [reply crypto/ocsp/ocsp_vfy.c:309]: I think in this function, it's -1 means error, 0 means no match, 1 means match.

Comment (paulidale) [reply crypto/ocsp/ocsp_vfy.c:309]: Comment added.

I also cleaned up the syntax and added additional errors on failure.

**Comment by paulidale** [comment]
Merged to master, thanks.

**Pull request closed without merging**